transformers/component_tests/initializers/events/event_initializers_test.go

```diff
@@ -131,6 +132,14 @@ var _ = Describe("Event transformer initializers", func() {
 		assertCorrectInit(transformer, name, topic, 8928165, catContracts())
 	})
 
+	It("configures clip kick", func() {
+		transformer := clip_kick.EventTransformerInitializer(db)
+		name := "clip_kick"
+		topic := "0x898eb267e6ff99efebaf458079f659c812f378fe8cf2a33ab4028717672bd021"
```

Comment:
Just for my own understanding, which components here are real and which are fake? Looks like this is a real topic, discovered by running the contracts locally, but the address and block are faked?

Reply:
That's right.. Those are to be replaced with real values when available on `mainnet`.  I was trying to make them look really fake 🤣